### PR TITLE
bug 1470496 - use a single gpg key across all prod scriptworkers

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ tasks:
           - master
     payload:
       maxRunTime: 3600
-      image: python
+      image: escapewindow/cot-gpg-keys
```
Comment (JohanLorenzo) [inline .taskcluster.yml:15]: Nit: We should move this docker image to https://hub.docker.com/u/mozillareleng/, in order to avoid permissions issues.

**Review by JohanLorenzo — APPROVED** [review]
LGTM!

File: .taskcluster.yml (hunk 1 above)
Comment (escapewindow) [reply .taskcluster.yml:15]: +1. I was putting that in the future-wontfix pile if and when we move off gpg keys, but it might be worth doing anyway.

**Comment by escapewindow** [comment]
Filed #51 for that.

**Pull request merged**